Gemfile

```diff
@@ -94,6 +94,10 @@ gem 'bootsnap', require: false
 
 gem 'font-awesome-sass', '~> 5.12.0'
 
+gem 'bcrypt'
```

Comment:
I think bcrypt is added by default in rails, no need to add explicitly, can you please verify whether this is needed?